.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,72 @@
+<!--  All Submissions -->
+
+
+## Community Note
+<!-- Please leave the community note as is. -->
+* Please vote on this PR by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original PR to help the community and maintainers prioritize for review
+* Please do not leave "+1" or "me too" comments, they generate extra noise for PR followers and do not help prioritize for review
+
+
+## PR Checklist
+
+- [ ] I have followed the guidelines in our [Contributing Documentation](../contributing/README.md).
+- [ ] I have checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change.
+- [ ] I have used a meaningful PR description to help maintainers and other users understand this change and help prevent duplicate work.
+Example:
+“`resource_name_here` - description of change e.g. adding property `new_property_name_here`”
+- [ ] I have checked if my changes close any open issues. If so please include appropriate [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) below.
+
+
+<!-- You can erase any parts of this template below this point that are not applicable to your Pull Request. -->
+
+
+## New Feature Submissions
+
+- [ ] My submission includes Test coverage as described in the [Contribution Guide](../contributing/topics/guide-new-resource.md) and the tests pass. (if this is not possible for any reason, please include details of why below)
+
+
+## Changes to existing Resource / Data Source
+
+- [ ] I have added an explanation of what my changes do and why I'd like you to include them (This may be covered by linking to an issue above, but may benefit from additional explanation).
+- [ ] I have written new tests for my resource or datasource changes.
+- [ ] I have successfully run tests with my changes locally. If not, please provide details on testing challenges that prevented you running the tests.
+
+
+## Documentation Changes
+
+- [ ] Documentation is written in International English.
+- [ ] Documentation is written in a helpful and kind way to assist users that may be unfamiliar with the resource / data source.
+
+
+## Description
+
+<!-- Please include a description below with the reason for the PR, what it is doing, what it is trying to accomplish, and anything relevant for a reviewer to know. It also helps to paste the output from running the acceptance tests. -->
+
+
+
```

Comment:
I think it could be helpful to have a separate heading underneath, specifically for `## Testing Evidence` in order to emphasize the importance of tests/testing for PRs.

From observation I've found that it's easy to forget adding test coverage for changes made to a resource/data source so this will hopefully serve as a strong reminder. This will also let us call out special testing requirements for specific changes, like state migrations.

```suggestion
## Description

<!-- Please include a description below with the reason for the PR, what it is doing, what it is trying to accomplish, and anything relevant for a reviewer to know. -->

## Testing Evidence

<!-- Please include your testing evidence here or an explanation on why no testing evidence can be provided. 

For state migrations please test the changes locally and provide details here, such as the versions involved in testing the migration path. For further details on testing state migration changes please see our guide on [state migrations](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/guide-state-migrations.md#testing) in the contributor documentation. -->

```